.github/workflows/build_docker.yml

```diff
@@ -0,0 +1,20 @@
+name: Build Docker
+
+on:
+  pull_request:
+    paths:
```

Comment:
Just checking we really want to test the image only when we change these files

Reply:
I'd say so because the action takes a long time and shouldn't really fail unless we change those files. The actual Rust part is checked by the regular matrix, but I'm open to suggestions. The release workflow also builds the image again.